### DISCUSSION TITLE
Support sizeof() of class/struct computation

### DISCUSSION THREAD

**Discussion by cacosub7** [clangd/clangd#1443] [Q&A]
Does clangd have a feature that evaluating the sizeof(class/struct) result on hover？ ![image](https://user-images.githubusercontent.com/46550268/209760467-0cbef5cd-0d9c-412f-9a43-0edfa6455a41.jpeg)
(this is what vscode C/C++ extension shows)

**Comment by HighCommander4** [comment]
Clangd supports this as well:

![Screenshot_20221228_013723](https://user-images.githubusercontent.com/1751085/209769426-4af1c37b-f7d0-4060-a1c6-ad2342877649.png)

**Reply by cacosub7** [reply]
Thanks, i found that my workspace directory are symbol linked, which causes lot of issues.
